lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
+
+  form > div {
+    position: relative;
+  }
+
+  .search-field {
+    @include box-sizing(border-box);
+    @include border-radius(4px);
+    top: 5px;
+    width: 100%;
+    background: $white-t1;
+    font-family: $sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    letter-spacing: normal;
+    &.is-active {
+      background: $white;
+    }
+  }
+
+  .search-button, .cancel-button {
+    @include box-sizing(border-box);
+    display: block;
+    position: absolute;
+    top: 0;
+    @include right(12px);
+    height: 35px;
+    padding: 0;
+    color: #888;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 35px;
+    border: none;
+    box-shadow: none;
+    text-shadow: none;
+    text-transform: none;
+    letter-spacing: normal;
+    background: transparent;
+  }
+
+  .cancel-button {
+    display: none;
+  }
+
+}
+
+
+.search-results {
+
+  display: none;
+
+  .search-info {
+    padding-bottom: $baseline;
+    border-bottom: 4px solid $border-color-l4;
+    margin-bottom: $baseline;
+    h2 {
+      font-family: $sans-serif;
+    }
+    .search-count {
+      @include float(right);
+      color: $gray-l1;
+    }
+  }
+
+  .search-result-list {
+    padding: 0;
+  }
+
+  .search-results-item {
+    position: relative;
+    border-bottom: 1px solid $gray-l4;
+    list-style-type: none;
+    margin-bottom: $baseline;
+    padding-bottom: $baseline;
+    @include padding-right(140px);
+
+    .sri-excerpt {
+      margin-bottom: $baseline;
+    }
+    .sri-type {
+      position: absolute;
+      @include right(0);
+      bottom: $baseline;
+      color: $gray-l1;
+      font-size: 14px;
+    }
+    .sri-course-name {
+      @include margin-right(1em);
+      color: $gray-l1;
+      font-size: 14px;
+    }
+    .sri-location {
+      color: $gray-l1;
+      font-size: 14px;
+    }
+    .sri-link {
+      position: absolute;
+      @include right(0);
+      top: 0;
+      line-height: 1.6em;
+      text-transform: uppercase;
+    }
+  }
+
+  .search-load-next {
+    display: block;
+    text-transform: uppercase;
+    color: $base-font-color;
+    border: 2px solid $link-color;
+    @include border-radius(3px);
+    padding: 1rem;
+  }
+
+}
+
+
+#courseware-search-bar  {
+  box-shadow: 0 1px 0 #fff inset, 0 -1px 0 rgba(0, 0, 0, .1) inset;
```

Comment:
I would look at the $shadow rules in _variables.scss to see if you can simplify this rule or reuse an existing shadow.